templates/flutter/test/services/service_test.dart.twig

```diff
@@ -31,6 +32,18 @@ class MockClient extends Mock implements Client {
   ) async {
     return super.noSuchMethod(Invocation.method(#webAuth, [url]), returnValue: 'done');
   }
+
+  @override
+  Future<Response> chunkedUpload({
+    required String path,
+    required Map<String, dynamic> params,
+    required String paramName,
+    required String idParamName,
+    required Map<String, String> headers,
+    Function(UploadProgress)? onProgress,
+  }) async {
+    return super.noSuchMethod(Invocation.method(#chunkedUpload, [path, params, paramName, idParamName, headers]), returnValue: Response());
```

Comment:
I keep getting this error:

<img width="935" alt="image" src="https://github.com/appwrite/sdk-generator/assets/1477010/16c7644c-44cd-412e-8f58-94db8f12068f">

Reply:
Ahh...it's because im matching it wrong in the `when()`...